spec/doc_formatter_spec.rb

```diff
@@ -1,3 +1,5 @@
+sleep 3
```

Comment:
```suggestion
# This delay visualizes the effect of load time calculation.
# https://github.com/serpapi/turbo_tests/pull/8#issue-583037321
sleep 3
```

Reply:
Just to keep all in one place I think I should actually explain: since I was not sure how to illustrate the fact that the loading time gets aggregated across all the specs I just decided to add that delay, so it'd be visible in the final number at the end of the report. There's no practical need in that delay other then aforementioned demo purposes, so it could and probably should be deleted / replaced with some better way to tests the tests (pardon the pun 🤗) in the future.